### DISCUSSION TITLE
Built In Cookie settings

### DISCUSSION THREAD

**Discussion by Higorgx** [squidfunk/mkdocs-material#4294] [Q&A] [answered]
Hello,

I have a question about the built in cookie consent popup with mkdocs-material.  I followed the example in the documentation but the popup does not render.
 Here is the snippet of the code:
![image](https://user-images.githubusercontent.com/36903838/187189609-b0e1f700-ef43-41e3-be8e-b807e34800c4.png)

Could you please tell me what I am doing wrong?

**Comment by Fabio-Duppre** [comment]
Same problem here

**Reply by leandrosouzalsti** [reply]
Me too, I'm waiting for the answer

**Comment by squidfunk** [comment, marked as answer]
Make sure to disable your Adblocker and any other extensions related to cookie consent. Ideally, test in incognito mode.

**Reply by Higorgx** [reply]
we have tested in incognito mode, in several browsers (chrome, firefox, edge, brave, opera), as well as in windows 11 and windows 10 on several different computers. our test sector was not successful in rendering the cookie consent popup.

**Reply by Higorgx** [reply]
![image](https://user-images.githubusercontent.com/36903838/187417630-b62c7e8e-e53e-4caf-8931-368fd0d9ea69.png)
when we tested the same cookie consent on mkdocs-material-master it worked without problems.

**Reply by squidfunk** [reply]
Do you have [analytics](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#google-analytics) configured? Honestly, it's frankly impossible for me to debug the situation without having a minimal reproducible example. If you find that your issue is not solved, please provide such an example and attach it here as a zip file.

**Reply by Higorgx** [reply]
thanks for the help!
We ran into some problems with our server and for some reason we couldn't install the latest version of material-mkdocs, even when we specified version 8.4.2
